### PR TITLE
 Added notification call back

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
@amymullins @uppalk1 @srinia6 @brianparry @geoff-reid @BillyAbildgaard @RackHD/corecommitters

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

File: lib/graphs/install-suse-graph.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ module.exports = {
...
-        }
-    ]
+        },
+        {
+            label: 'rackhd-callback-notification-wait',
+            taskName: 'Task.Wait.Notification',
+            waitOn: {
+                'install-os': 'succeeded'
+            }
+        }]
```
Comment (anhou) [inline lib/graphs/install-suse-graph.js]: ']' in a new line is better

**Pull request merged**